modules/thermo-cycler/thermo-cycler-arduino/lid.h

```diff
@@ -74,7 +74,9 @@
 #define PULSE_HIGH_MICROSECONDS 2
 #define MOTOR_STEP_DELAY 60   // microseconds
 #define LID_CLOSE_BACKTRACK_ANGLE 3.0
```

Comment:
Maybe some comments about what these angles are for and when they're used?

Reply:
Pushed new commit and added the comments in the open & close functions since explaining what the angles are will need explanation of the whole open & close process